README.md

```diff
@@ -226,6 +226,11 @@ TODO: Move this code over to the idseq-dag repo.
 
 ## Release notes
 
+- 3.8.0
+   - Creates a [status name]_status.json file for each dag_json received, which each step updates with information
```

Comment:
Would it make sense to remove Emily's line here? (not sure how y'all plan to merge)

Reply:
We're merging both our PRs into the `3.8` branch, which we will then merge into `master` when ready to deploy alongside `idseq-web`

Reply:
Nvm, she went ahead and merged into master ahead of me.